README.md

```diff
@@ -20,8 +20,8 @@ Once a Mist version is released the Meteor frontend part is bundled using `meteo
 
 To run mist in development you need [Node.js NPM](https://nodejs.org) and [Meteor](https://www.meteor.com/install) installed. When this is done, install Electron:
 
-    $ npm install -g electron-prebuilt
     $ curl https://install.meteor.com/ | sh
+    $ sudo npm install -g electron-prebuilt
```

Comment:
good point, and I thought about that as I wrote that. It's common that -g flag requires sudo and the way to make -g install to local user space seemed... well not -g(lobal) 
That being said, I agree. It's too easy to set up users incorrectly and leave un necessary vulnerabilities that can be pushed to a prod env.